### ISSUE TITLE
Widescreen Hud Placement

### ISSUE THREAD

**Issue by nrb547** [multitheftauto/mtasa-blue#848]
On MTA the original HUD takes too much of your screen, even if you turn on the option in settings that should adjust the proportions of it. It's just way too big, you can see in this thread how the HUD looks like on singleplayer or samp with widescreen fix made by thirteenag. https://forum.mtasa.com/topic/103042-widescreen-fix-for-mta/?_fromLogin=1

**Comment by jushar** [comment]
For reference: https://github.com/ThirteenAG/WidescreenFixesPack/tree/master/source/GTASA.WidescreenFix

**Comment by LoneDev6** [comment]
@botder
I have this problem with ultrawide screen, huds are really huge
![image](https://user-images.githubusercontent.com/27242001/79689942-38ce5280-8258-11ea-93bd-302509f3daa0.png)

**Comment by qaisjp** [comment]
My suggestion would be to just enable this by default (maybe without even making a client setting - it could just be a CVAR), but allowing scripts to use the old HUD by using `setGlitchEnabled`.

**Comment by LoneDev6** [comment]
> 
> 
> My suggestion would be to just enable this by default (maybe without even making a client setting - it could just be a CVAR), but allowing scripts to use the old HUD by using `setGlitchEnabled`.

enabling what feature exactly?
I tried setting automatic hud and checking the aspect ratio thing with no luck, you can see the result on my screenshot.
A good thing would be to allow us to resize the hud with a custom percentage setting or something similar

**Comment by qaisjp** [comment]
> enabling what feature exactly?

enable the enhancement that this issue is tracking. it is a high-level suggestion on how we can implement this feature.

**Comment by Clippy95** [comment]
I'd like to also point out that the issue isn't just the HUD, the FOV is too low for 21:9+ as well.

**Comment by Dark-Dragon** [comment]
This honestly seems more like a preference-thing to me. The ideal solution would probably be a hud-size setting, with scripting functions to retrieve the current setting or possibly even adjust it for the the time the player is on the server. If that is feasible at all.

**Comment by rifleh700** [comment]
This is not only hud problem. Bug also affects another dx textures (lamps, sky, etc)
![screenshot](https://i.imgur.com/hBAMzzP.png)

**Comment by henrymarques** [comment]
![mtasa without widescreen fix](https://github.com/multitheftauto/mtasa-blue/assets/28879034/cddfb531-0871-4685-a5c9-9e983c855ad9)
![mtasa widescreen fix](https://github.com/multitheftauto/mtasa-blue/assets/28879034/567ded3e-035e-4ab6-ab75-f1eb1a0d1c05)
I got some progress

**Comment by bum8hj** [comment]
I could be wrong, but I'm pretty sure the weapon icon should be a square. It looks squeezed after your fix.

**Comment by henrymarques** [comment]
You're right, I just fix it. There still hooks and access violations to fix too. That was just some hours of work.

![squared icon](https://github.com/multitheftauto/mtasa-blue/assets/28879034/2de43a3a-9e18-49b8-92f4-9bfc14bf6090)

**Comment by Fernando-A-Rocha** [comment]
so this still needs to be fixed apparently
